### PR TITLE
india engineering team

### PR DESCRIPTION
**Pull request by joshua-levin-deliveroo**
This is not our usual content, but it's being done to assist the team in india who are about to go live tomorrow with publicity on the launch, but who don't have their recruitment microsite ready yet.

### CODE REVIEW

File: _posts/2022-03-07-deliveroo-engineering-in-india.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+The Engineering Centre in Hyderabad will be a core part of Deliveroo’s central technology organisation. The company has already started recruiting and plans to employ over 150 engineers in India from the fields of analytics, platforms, automation and machine learning by the end of 2022. The Engineering Centre will be Deliveroo’s largest technology hub outside the United Kingdom, where the company is headquartered.
+
+# Engineering team
+
+<figure>
+![Deliveroo Company Values](/images/posts/deliveroo-engineering-in-india/deliveroo_values.jpg)
+</figure>
+
+Deliveroo operates a three-sided marketplace made up of consumers, restaurants and grocers, and delivery riders to bring people food and groceries in as little as 20 minutes. Deliveroo’s service is underpinned by pioneering logistics technology based on intelligent machine learning. The launch of the India Engineering Centre provides unparalleled opportunities for engineers to solve cutting edge business problems as Deliveroo scales and enhances its offering.
```
Comment (rbedzkow) [inline _posts/2022-03-07-deliveroo-engineering-in-india.md:33]: Sorry to nitpick, but `cutting-edge` might be more appropriate here

Comment (joshua-levin-deliveroo) [reply _posts/2022-03-07-deliveroo-engineering-in-india.md]: changed!

File: _posts/2022-03-07-deliveroo-engineering-in-india.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+***Global food delivery platform aims to expand its world-class engineering capabilities with a new team of talent from across India***
+
+***New Engineering Centre launched in Hyderabad will employ over 150 engineers by the end of 2022 as part of a multi-year project***
+
+***Engineers will work on products for Deliveroo’s growing grocery service, supporting the rider network and improving the in-app customer experience***
+
+# India, March 09,2022
+
+Deliveroo, a global food delivery company operating across Europe, the Middle East, Asia, and Australia, announces the launch of its India Engineering Centre in Hyderabad.
```
Comment (Crunch09) [inline _posts/2022-03-07-deliveroo-engineering-in-india.md:21]: Not sure if we need this and similar parts? I understand its copied from a press release but I would expect someone who visits the deliveroo engineering blog to know what Deliveroo is.

**Review by Crunch09 — APPROVED** [review]
thanks!

**Review by rbedzkow — APPROVED** [review]
My editorial eye sees no issues

**Pull request merged**